### PR TITLE
release 1.0.0

### PR DESCRIPTION
**Pull request by evgeni**

### CODE REVIEW

**Comment by evgeni** [comment]
draft, as I want to get #73 into the release

**Review by mdellweg — APPROVED** [review]

File: changelogs/changelog.yaml [hunk 1]
```diff
@@ -6,3 +6,10 @@ releases:
     release_date: '2021-02-11'
   0.3.0:
     release_date: '2021-04-06'
+  1.0.0:
+    changes:
+      release_summary: 'This is the first stable release of the ``theforeman.operations``
+        collection.
+
+        '
```
Comment (ehelms) [inline changelogs/changelog.yaml]: This block of text here looks weird to me

Comment (evgeni) [reply changelogs/changelog.yaml]: that's how `antsibul-changelog` generates it. :woman_shrugging:

Comment (evgeni) [reply changelogs/changelog.yaml]: (fixed)

**Review by ehelms — APPROVED** [review]

**Pull request merged**